sklearn/cluster/_hierarchical_fast.pyx

```diff
@@ -8,9 +8,6 @@ ctypedef np.float64_t DOUBLE
 ctypedef np.npy_intp INTP
 ctypedef np.int8_t INT8
 
-# Numpy must be initialized. When using numpy from C or Cython you must
-# _always_ do that, or you will have segfaults
-
```

Comment:
This is a naive question: why is this comment removed? Is this related to the new common directives setup?

Reply:
It's not related to the directives. I removed it because we have this numpy.import_array in all cython files but this comment is in this file only. It could maybe belong to a developer doc but not here to me.

Reply:
I see. I also think it's fine removing it (this is kind of implicit when using numpy via Cython).